ci/terraform/dev.tfvars

```diff
@@ -45,3 +45,5 @@ orch_stub_to_auth_client_id          = "orchestrationAuth"
 orch_stub_to_auth_audience           = "https://signin.dev.account.gov.uk/"
 
 dynatrace_secret_arn = "arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables"
+
+support_multi_channel = "1"
```

Comment:
Do we also need to update this for the new frontend?

Reply:
Hi @ethanmills - with help from @alhcomer I've located the relevant line in the CloudFormation template and switched it to match (in the same commit).